### PR TITLE
Combine the variable section in e2e and also introuce new vars.

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 PTAL .

File: e2e/rbd.go [hunk 1]
```diff
@@ -12,12 +12,11 @@ var (
...
-)
-
-var (
-	rbdDirPath = "../deploy/rbd/kubernetes/"
-
-	rbdExamplePath = "../examples/rbd/"
+	rbdDirPath         = "../deploy/rbd/kubernetes/"
+	rbdExamplePath     = "../examples/rbd/"
+	rbdDeploymentName  = "csi-rbdplugin-provisioner"
+	rbdDaemonsetNamr   = "csi-rbdplugin"
```
Comment (Madhu-1) [inline e2e/rbd.go]: rbdDaemonsetNamr/rbdDaemonsetName

Comment (humblec) [reply e2e/rbd.go]: @Madhu-1 done.thanks

**Comment by Madhu-1** [comment]
@humblec CI failure
```
e2e/rbd.go:19:2: `namespace` redeclared in this block (typecheck)
	namespace          = "default"
	^
e2e/cephfs.go:18:2: 	other declaration of namespace (typecheck)
	namespace             = "default"
	^
```

**Comment by humblec** [comment]
> @humblec CI failure
> 
> ```
> e2e/rbd.go:19:2: `namespace` redeclared in this block (typecheck)
> 	namespace          = "default"
> 	^
> e2e/cephfs.go:18:2: 	other declaration of namespace (typecheck)
> 	namespace             = "default"
> 	^
> ```

Removed from rbd. PTAL

**Pull request merged**